external_dns.tf

```diff
@@ -39,6 +39,7 @@ locals {
 }
 
 resource "helm_release" "external_dns" {
+  count           = var.enable_external_dns ? 1 : 0
```

Comment:
Should we also specify this conditional install for the above `module "external_dns_sa"`?

Reply:
See earlier comment